flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -25,9 +25,10 @@ class FlxPexParser
 	 * @param	data			The data to be parsed. It has to be an ID to the assets file, a file embedded with @:file(), a string with the content of the file or a XML object.
 	 * @param	particleGraphic	The particle graphic
 	 * @param	emitter			(optional) A FlxEmitter. Most properties will be overwritten!
+	 * @param	scale			(optional) Used to scale the resulting emitter. Will scale both the texture size and positions of resulting particles.
 	 * @return	A new emitter	
 	 */
-	public static function parse<T:FlxEmitter>(data:Dynamic, particleGraphic:FlxGraphicAsset, ?emitter:T):T
+	public static function parse<T:FlxEmitter>(data:Dynamic, particleGraphic:FlxGraphicAsset, ?emitter:T, ?scale:Float=1):T
```

Comment:
I don't think `scale` should have a `?`, that unnecessarily makes it a `Null<Float>` (see my SO answer [here](http://stackoverflow.com/questions/31914614/is-question-mark-operator-in-haxe-really-necessary/31916466#31916466)). Or am I missing something?

Also, since you're already at it: Flixel's code style has spaces before and after the `=`.